mission_planner/mInterestZone interest_zone
int8 ID0
int8 ID1
int8 ID2
int8 ID3